build.sbt

```diff
@@ -61,17 +58,13 @@ libraryDependencies ++= Seq(
   "com.softwaremill.macwire"  %% "macrosakka"                    % "2.3.7" % "provided",
   "com.softwaremill.macwire"  %% "proxy"                         % "2.3.7",
   "com.softwaremill.macwire"  %% "util"                          % "2.3.7",
-  "org.typelevel"             %% "cats-core"                     % "1.6.1",
-  "uk.gov.hmrc"               %% "auth-client"                   % "3.0.0-play-26",
-  "uk.gov.hmrc"               %% "bootstrap-backend-play-26"     % "2.24.0",
-  "uk.gov.hmrc"               %% "mongo-lock"                    % "6.23.0-play-26",
-  "uk.gov.hmrc"               %% "simple-reactivemongo"          % "7.30.0-play-26",
-  "org.scala-stm"             %% "scala-stm"                     % "0.9.1"
-)
-
-resolvers ++= Seq(
-  Resolver.bintrayRepo("hmrc", "releases"),
-  Resolver.jcenterRepo
+  "org.typelevel"             %% "cats-core"                     % "2.4.0",
+  "uk.gov.hmrc"               %% "bootstrap-backend-play-26"     % "5.3.0",
```

Comment:
We can use `5.4.0` for bootstrap-backend-play-26. They did one small improvement in this release.